README.md

```diff
@@ -16,7 +16,11 @@ and hence are supposed to be updated there (whenever needed) and synced into thi
 
 ## Syncing the files from Keycloak repository
 
-* Checkout [main Keycloak server repository](https://github.com/keycloak/keycloak) and build it on your laptop to make sure latest Keycloak stuff available in your local maven repository.
+* Fetch [main Keycloak server repository](https://github.com/keycloak/keycloak) and checkout the last `release/X.Y` branch (For example `git checkout release/26.0` ). Note that we usually
```

Comment:
Let's not add line breaks in markdown files in the middle of sentences, line wrapping is handled automatically.

Reply:
Line wrapping is handled automatically in the resulting HTML, however in some text editors (at least in the Intellij idea), it is not handled automatically and it results in one huge line, which is harder to update and deal with...

I see we use line wrapping on some other places in Keycloak like for example here https://github.com/keycloak/keycloak/blob/main/CONTRIBUTING.md?plain=1#L13 and even if line is wrapped, the resulting HTML looks the same and it renders it the same way like single line https://github.com/keycloak/keycloak/blob/main/CONTRIBUTING.md .

Considering this, my preference is still to rather use line wrapping as it doesn't impact resulting HTML, but is easier to edit IMO. But if you strongly feel otherwise, I can update and/or we can start discussion with the wider team as it is currently used on many places. WDYT?

Reply:
My mistake, I was not aware this wrapping would be handled correctly. Consider this resolved. FYI you can toggle line-wrapping in IntellJ pretty easily.